apps/api/src/utils/permissions/resolve-roles.ts

```diff
@@ -38,5 +41,15 @@ export function getPermissions(
     ...out,
     ...resolvePermissionsforUser(context.user),
   ];
+
+  if (context.agentRegistrationId) out = [
+    ...out,
+    permissions.activeAgent.internal.register({}),
+  ];
+  if (context.activeAgentId) out = [
+    ...out,
+    permissions.activeAgent.internal.manage({ id: context.activeAgentId }),
+  ];
```

Comment:
(nit) For multiline statements, I'd prefer we put braces around it i.e.


```suggestion
  if (context.agentRegistrationId) {
    out = [
      ...out,
      permissions.activeAgent.internal.register({}),
    ];
  }
  if (context.activeAgentId) {
    out = [
      ...out,
      permissions.activeAgent.internal.manage({ id: context.activeAgentId }),
    ];
  }
```

Reply:
I redid the function a little with `.concat`, looks better now